docs/contributing.md

```diff
@@ -23,10 +19,11 @@ Before starting to write code and opening a pull request, please take a look [at
 
 Pull requests should always belong to an [issue in our issue tracker](#bug-reporting).
 
-To merge a pull request, the following is necessary:
-* All checks (linting, flow, unit and e2e tests) have to pass.
+To merge a pull request, it has to match our Definition of Done. It includes among others:
+* All checks (linting, flow, unit and e2e tests, ...) have to pass.
 * No changes are requested.
 * Two approvals are needed.
+* New and changed functionality should be tested sufficiently.
```

Comment:
Maybe we can add here, that a test (if possible) should be written for the feature and a release note (if visible changes for end users?)

Reply:
Not sure what you want me to write here to be honest :D I am already writing here that functionality has to be tested... Release notes are mentioned five lines below.

Reply:
i unterstood your "should be tested" as a manual test and thought it would be good to add, that a (unit) test should also be added, but nevermind ;)

Reply:
Ah I see. Yeah will add something here.